gprt/gprt.cpp

```diff
@@ -3175,6 +3179,15 @@ GPRT_API void gprtContextDestroy(GPRTContext _context)
   LOG("context destroyed...");
 }
 
+GPRT_API void
+gprtContextSetRayTypeCount(GPRTContext _context,
```

Comment:
Should there be a getter for this property too?

Reply:
ah, good idea. We could add this